pkg/services/auth/auth.go

```diff
@@ -124,24 +120,29 @@ func (a *authSvc) CreateGitClient(ctx context.Context, repoUrl gitproviders.Repo
 	return git.New(pubKey, wrapper.NewGoGit()), nil
 }
 
-// setupDeployKey creates a git.Git client instrumented with existing or generated deploy keys.
+// SetupDeployKey creates a git.Git client instrumented with existing or generated deploy keys.
 // This ensures that git operations are done with stored deploy keys instead of a user's local ssh-agent or equivalent.
-func (a *authSvc) setupDeployKey(ctx context.Context, name SecretName, targetName string, repo gitproviders.RepoURL) (*ssh.PublicKeys, error) {
+func (a *authSvc) SetupDeployKey(ctx context.Context, namespace string, repo gitproviders.RepoURL) (*ssh.PublicKeys, error) {
```

Comment:
Wasn't this changed in another branch? I have the feeling that some stuff from other branches made it onto this PR.

Reply:
I'm can't tell if there are any other branches with this change. I intentionally changed this 🙂